invenio_config_tugraz/templates/invenio_app_rdm/help/search.de.html

```diff
@@ -0,0 +1,257 @@
+      <strong>Beispiel:</strong>
+      <a href="/search?page=1&amp;size=20&amp;q=metadata.title:(open%20science%29"
+        ><code>metadata.title:(open science)</code></a
+      >
+    </p>
+    <p>
+      Die vollständige Liste der Felder, die Sie durchsuchen können, finden Sie in der Feldreferenz unten.
+    </p>
+    <h3>Kombinierte einfache, Phrasen- oder Feldsuche</h3>
+    <p>
+      <strong>Beispiel:</strong>
+      <a
+        href="/search?page=1&amp;size=20&amp;q=%2Bmetadata.title:%22open%20science%22%20-metadata.title:policy"
+        ><code>+metadata.title:"open science" -metadata.title:policy</code></a
+      >
+      oder z. B.
+      <a href="/search?page=1&amp;size=20&amp;q=metadata.title:(-open%20%2Bscience%29"
+        ><code>metadata.title:(-open +science)</code></a
+      >
+    </p>
+    <p>
+      Sie können einfache, Phrasen- und Feldsuche kombinieren, um erweiterte Suchanfragen zu konstruieren.
+    </p>
+    <h3>Bereichssuche</h3>
+    <p>
+      <strong>Beispiel:</strong>
+      <a
+        href="/search?page=1&amp;size=20&amp;q=metadata.publication_date:%5B2017%20TO%202018%5D"
+        ><code>metadata.publication_date:[2017 TO 2018]</code></a
+      >
+      (Achtung, <code>TO</code> muss großgeschrieben werden).
+    </p>
+    <p>
+      Die Ergebnisse entsprechen allen Datensätzen mit einem Veröffentlichungsdatum
+      zwischen 2017-01-01 und 2018-01-01 (beide Daten inklusive).
+    </p>
+    <p>Beachten Sie, dass partielle Daten zu vollständigen Daten erweitert werden, z. B.:</p>
+    <ul>
+      <li>2017 wird erweitert auf 2017-01-01</li>
+      <li>2017-06 wird erweitert auf 2017-06-01</li>
+    </ul>
+    <p>
+      Verwenden Sie eckige Klammern (<code>[]</code>) für <em>einschließende</em> Bereiche und geschweifte
+      Klammern (<code>{}</code>) für <em>ausschließende</em> Bereiche, z. B.:
+    </p>
+    <ul>
+      <li>
+        <code>[2017 TO 2018}</code> ist aufgrund der Datumserweiterung
+        und der exklusiven Obergrenze äquivalent zu
+        <code>[2017-01-01 TO 2017-12-31]</code>.
+      </li>
+    </ul>
+    <p>Beispiele für andere Bereiche:</p>
+    <ul>
+      <li>
+        <code>metadata.publication_date:{* TO 2017-01-01}</code>: Alle Tage bis 2017.
+      </li>
+      <li>
+        <code>metadata.publication_date:[2017-01-01 TO *]</code>: Alle Tage von 2017.
+      </li>
+    </ul>
+    <h3>Ranking/Sortierung</h3>
+    <p>
+      Standardmäßig werden alle Suchen nach einem internen Ranking-Algorithmus sortiert,
+      der jede Übereinstimmung mit Ihrer Abfrage bewertet. Sowohl in der Benutzeroberfläche
+      als auch in der REST-API ist es möglich, die Ergebnisse zu sortieren nach:
+    </p>
+    <ul>
+      <li>Aktuellste</li>
+      <li>Beste Übereinstimmung</li>
+    </ul>
+    <h3>Reguläre Ausdrücke</h3>
+    <p>
+      Reguläre Ausdrücke sind eine mächtige Sprache für den Mustervergleich, mit der
+      man nach bestimmten Mustern in einem Feld suchen kann. Wenn wir zum Beispiel
```

Comment:
Here we start to talk about 'wir' (we) while beforehand it was 'unpersonal' (one can) or 'Sie' (formal you). It feels a bit like a style break.